apps/api/scripts/seed-database.js

```diff
@@ -527,13 +634,51 @@ Rx.of({})
       }
       return Rx.from(matchesFlat)
     }),
-    concatMap(redMatchCreate)
+    concatMap(redMatchCreate),
+    tap(() => console.log('--- DONE Create Match Mentor-Mentee Pairs ----')),
```

Comment:
It seems that this code is not executed, as I don't see log entries. Need to fix. My idea is to extract this and place into a separate file, as e.g. `create-roles.js` is done